wordpress-mobile/gutenberg-mobile
# Fix caret lost regression

Fixes #1031 , #1034 .

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/15833

Fix for the regression where the caret is lost on paragraph split.

To test:

1. Run the demo app and place the caret in the middle of a paragraph block
2. Tap "Enter" to split the block into two
3. The block should split as normal and the caret be placed at the start of the new paragraph block. Note: #1029 still applies so, the virtual keyboard will be closed.

Update release notes:
